automobile/sumo-interface.md

```diff
@@ -4,79 +4,131 @@ An interface with a microscopic traffic simulator called `SUMO` (Simulation of
-
-%end
 
+### The SumoInterface PROTO
 
-> **Note** [macOS]:
-On macOS, SUMO relies on X11. You need therefore to install [XQuartz](https://www.xquartz.org) (version 2.7.8 or later) for the interface to work.
+Here is the parameters of the `SumoInterface` PROTO (which inherits from the `Supervisor` node):
+
+```
+PROTO SumoInterface [
+  field SFBool    gui                   TRUE
+  field SFBool    useNetconvert         TRUE
+  field SFBool    enableTrafficLights   TRUE
+  field SFBool    enableWheelsRotation  FALSE
+  field SFInt32   maxVehicules          100
+  field SFInt32   radius               -1
+  field SFFloat   maximumLateralSpeed   2.5
+  field SFFloat   maximumAngularSpeed   3
+  field SFFloat   laneChangeDelay       3
+  field SFBool    enableHeight          FALSE
+  field SFInt32   step                  200
+  field SFInt32   port                  8873
+  field SFInt32   seed                  1
+  field SFString  networkFiles          ""
+  field SFString  sumoArguments         ""
+  field SFNode    display               NULL
+  field SFBool    verbose               FALSE
+  field SFBool    synchronization       TRUE
+  field MFNode    children              []
+]
+```
 
+#### SumoInterface fields summary
+
+- `gui`: Defines if the command-line or GUI version of SUMO should be used.
+- `useNetconvert`: Defines if NETCONVERT should be called before launching SUMO, this should be disabled if a `*.net` is already provided.
+- `enableTrafficLights`: Defines if the traffic lights synchronization should be used or not.
+- `enableWheelsRotation`: Defines if the wheels of the vehicles should rotate or not (wheels rotation has an impact on the simulation speed but make the simulation looks more realistic).
+- `maxVehicules`: Defines the maximum number of vehicles added in Webots from SUMO.
+- `radius`: Defines the visibility radius of the vehicles in meters (only vehicles closer to the viewpoint than this radius will be added in Webots). A negative value means that all the vehicles are added.
+- `maximumLateralSpeed`: Defines the maximal lateral speed of any vehicle in meter per second.
+- `maximumAngularSpeed`: Defines the maximal angular speed of any vehicle in radian per second.
+- `laneChangeDelay`: Defines the time spent to change lane (during this period, the position of the vehicle in Webots and SUMO may not be synchronized anymore).
+- `enableHeight`: Defines whether height information should be extracted from the edge name (experimental).
+- `step`: Defines the time step of SUMO in milliseconds (should be bigger or equal to the world time step).
+- `port`: Defines which port SUMO and Webots should use to communicate.
+- `seed`: Defines the seed of the SUMO random number generator. Use `0` for a random seed (`--random` option of SUMO).
```

Comment:
I guess 0 corresponds to a time-based random seed, right? If so, that should documented more precisely.

Reply:
Yes it is, but on our side what we exactly do is only to send the `--random` argument to SUMO at startup, then SUMO internally uses time-based randomness, but I was not sure we want to document this as this should be documented in the SUMO documentation and maybe not in our.

Reply:
Yes, but that sentence sounds wrong, because when we use `0`, it is not a random seed, but rather a time-based seed (non reproducible).